schemas/com.snowplowanalytics.snowplow/media_player_event_ad_break_end/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event that signals the end of an ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
```

Comment:
What are your thoughts on changing the vendor here (similar to what we do for mobile) to `com.snowplowanaytics.snowplow.mediaplayer`?

Reply:
I like that! I see that ecommerce schemas follow a similar pattern. I will change it, but I think I prefer `com.snowplowanaytics.snowplow.media`. I will also rename the event names to drop the `media_player` part.

Reply:
✅